percy-capybara.gemspec

```diff
@@ -18,11 +18,11 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ['lib']
 
-  spec.add_dependency 'percy-client', '~> 1.12'
+  spec.add_dependency 'percy-client', '~> 1.13'
```

Comment:
`percy-client` 1.13 is required. This PR is not compatible with earlier versions.